assets/chezmoi.io/docs/reference/templates/1password-functions/onepasswordRead.md

```diff
@@ -1,8 +1,8 @@
-# `onepasswordRead` *url* [*account*]
+# `onepasswordRead` _url_ [*account*]
 
 `onepasswordRead` returns data from [1Password](https://1password.com/) using
-the [1Password CLI](https://developer.1password.com/docs/cli) (`op`). *url* is
-passed to `op read $URL`. If *account* is specified, the extra arguments
+the [1Password CLI](https://developer.1password.com/docs/cli) (`op`). _url_ is
+passed to the `op read` command. If _account_ is specified, the extra arguments
```

Comment:
nit: it looks like `*word*` got reformatted to `_word_`. Could you undo this please?

Reply:
Good catch. I always forget to `:au! neoformat_config` when editing markdown in this repo.